dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -1268,6 +1272,11 @@ public static boolean isStaticColumn( String propName )
                 .anyMatch( orderColumn -> orderColumn.getPropName().equals( propName ) );
         }
 
+        public static boolean isFieldEqualToEnrolledAt( String fieldName )
```

Comment:
If you make this a method on OrderColumn (non-static) than it can be reused for other values on the enum. For example like `ENROLLED_AT.isProperty("enrolledat")`. You could also add one for the column if we are also comparing to that `isColumn()`